pipeline-templates/job/code-build.yml

```diff
@@ -1,18 +1,19 @@
-parameters:
-  SolutionBaseName:
-  BuildConfiguration:
-
 jobs:
 - job: CodeBuild
+  pool:
+    name: DAS - Continuous Integration
   variables:
   - group: BUILD Management Resources
+  - name: SolutionBaseName
+    value: 'SFA.DAS.Roatp.Functions' 
+
   steps:
-    - task: gitversion/setup@0
+    - task: gittools.gittools.setup-gitversion-task.gitversion/setup@0

```

Comment:
The task can remain as the shortened gitversopm/setup@0 as in the das-github-tempate